django/template/backends/dummy.py

```diff
@@ -60,4 +66,4 @@ def render(self, context=None, request=None):
         if request is not None:
             context['csrf_input'] = csrf_input_lazy(request)
             context['csrf_token'] = csrf_token_lazy(request)
-        return self.safe_substitute(context)
+        yield self.safe_substitute(context)
```

Comment:
I don't find it acceptable to fake streaming behavior and silently return a single chunk in a backend that's supposed to be a template for third-party backends. I think you should raise an exception if `stream=True`.